pkgs/by-name/ae/aegisub/package.nix

```diff
@@ -53,34 +60,74 @@ let
     CoreFoundation
     CoreText
     IOKit
-    OpenAL;
+    OpenAL
+    QuartzCore;
+
+  bestsource = fetchFromGitHub {
+    owner = "vapoursynth";
+    repo = "bestsource";
+    # https://github.com/arch1t3cht/Aegisub/blob/feature_11/subprojects/bestsource.wrap#L3
+    rev = "ba1249c1f5443be6d0ec2be32490af5bbc96bf99";
+    hash = "sha256-9BnyRzF33otju3W503O18JuTyvp+hFxk6JMwrozKoZY=";
+  };
+
+  vapoursynth = fetchFromGitHub {
+    owner = "vapoursynth";
+    repo = "vapoursynth";
+    # https://github.com/arch1t3cht/Aegisub/blob/feature_11/subprojects/vapoursynth.wrap#L3
+    rev = "R59";
+    hash = "sha256-6w7GSC5ZNIhLpulni4sKq0OvuxHlTJRilBFGH5PQW8U=";
+  };
+
+  AviSynthPlus = fetchFromGitHub {
+    owner = "AviSynth";
+    repo = "AviSynthPlus";
+    # https://github.com/arch1t3cht/Aegisub/blob/feature_11/subprojects/avisynth.wrap#L3
+    rev = "v3.7.2";
+    hash = "sha256-PNIrDRJNKWEBPEKlCq0nE6UW0prVswE6mW+Fi4ROTAc=";
+    fetchSubmodules = true;
+  };
+
+  gtest = fetchurl {
+    # https://github.com/arch1t3cht/Aegisub/blob/feature_11/subprojects/gtest.wrap#L4
+    url = "https://github.com/google/googletest/archive/release-1.8.1.zip";
+    hash = "sha256-kngnwYPQFzTMXP74Xg/z9akv/mGI4NGOkJxe/r8ooMc=";
+  };
+
+  gtest_patch = fetchurl {
+    # https://github.com/arch1t3cht/Aegisub/blob/feature_11/subprojects/gtest.wrap#L8
+    url = "https://wrapdb.mesonbuild.com/v1/projects/gtest/1.8.1/1/get_zip";
+    hash = "sha256-959f1G4JUHs/LgmlHqbrIAIO/+VDM19a7lnzDMjRWAU=";
+  };
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "aegisub";
-  version = "3.3.3";
+  version = "feature_11";
 
   src = fetchFromGitHub {
-    owner = "wangqr";
-    repo = "aegisub";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-oKhLv81EFudrJaaJ2ga3pVh4W5Hd2YchpjsoYoqRm78=";
+    owner = "arch1t3cht";
+    repo = "Aegisub";
+    rev = finalAttrs.version;
+    hash = "sha256-yEXDwne+wros0WjOwQbvMIXk0UXV5TOoV/72K12vi/c=";
   };
 
   nativeBuildInputs = [
+    meson
     cmake
```

Comment:
Is cmake needed?

Reply:
Yes, build avisynth requires cmake

Reply:
Then set `dontUseCmakeConfigure = true;`, since it is obvious that Meson is the main system.